mongorestore/restore.go

```diff
@@ -25,6 +25,33 @@ import (
 
 const insertBufferFactor = 16
 
+var indexOptions = map[string]bool{
```

Comment:
Please add a comment about which source file this is taken from.  Also, please rename to something like `validIndexOptions` for clarity.